### PR TITLE
Setup project folder and add migrations

### PR DESCRIPTION
**Pull request by SergSkol**
-Setup folder
-Add migrations

### CODE REVIEW

**Comment by prolajumokeoni** [comment]
Hi @SergSkol, 
While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have a professional and descriptive readme
kindly add a descriptive readme, please use the updated microverse readme template 
Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: db/migrate/20230228084538_add_user_ref_to_posts.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class AddUserRefToPosts < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :posts, :user, null: false, foreign_key: true
+  end
```
Comment (thecodechaser) [inline db/migrate/20230228084538_add_user_ref_to_posts.rb]: - [ ] The user foreign key for other tables should be as `author_id` not as `user_id` to follow the entities as on the ERD diagram. Please make sure you follow this and I would suggest looking at the below code for some hints.

```ruby
add_reference :posts, :author, null: false, foreign_key: {to_table: 'users'}

File: db/migrate/20230228084941_add_user_ref_to_likes.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class AddUserRefToLikes < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :likes, :user, null: false, foreign_key: true
+  end
```
Comment (thecodechaser) [inline db/migrate/20230228084941_add_user_ref_to_likes.rb]: - [ ] The user foreign key for other tables should be as `author_id` not as `user_id` to follow the entities as on the ERD diagram. Please make sure you follow this and I would suggest looking at the below code for some hints.

```ruby
add_reference :likes, :author, null: false, foreign_key: {to_table: 'users'}

File: db/migrate/20230228085212_add_user_ref_to_comments.rb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+class AddUserRefToComments < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :comments, :user, null: false, foreign_key: true
+  end
```
Comment (thecodechaser) [inline db/migrate/20230228085212_add_user_ref_to_comments.rb]: - [ ] The user foreign key for other tables should be as `author_id` not as `user_id` to follow the entities as on the ERD diagram. Please make sure you follow this and I would suggest looking at the below code for some hints.

```ruby
add_reference :comments, :author, null: false, foreign_key: {to_table: 'users'}

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @SergSkol👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- No linter errors. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: db/migrate/20230228084538_add_user_ref_to_posts.rb (hunk 1 above)
Comment (SergSkol) [reply db/migrate/20230228084538_add_user_ref_to_posts.rb]: Thanks!

File: db/migrate/20230228084941_add_user_ref_to_likes.rb (hunk 2 above)
Comment (SergSkol) [reply db/migrate/20230228084941_add_user_ref_to_likes.rb]: Thanks!

File: db/migrate/20230228085212_add_user_ref_to_comments.rb (hunk 3 above)
Comment (SergSkol) [reply db/migrate/20230228085212_add_user_ref_to_comments.rb]: Thanks!

**Review by AmaduKamara — APPROVED** [review]
# Status: Approved 🟢 🟢 🟢

Hi @SergSkol,

Good job working on the changes highlighted by the previous reviewer 👍🏽

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! 🎉 

## Highlights:
✅ You used Gitflow correctly 👍🏽
✅ Linter checks passed 👍🏽
✅ Good commit messages 👍🏽
✅ Create data model as required 👍🏽

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**